content/en/docs/security/_index.md

```diff
@@ -0,0 +1,60 @@
+
+Note that while Flux is very active it is a vendor-neutral CNCF project maintained by volunteers, not by a single company. As such,  security issue handling is done on a best-effort basis.
+
+## Controller permissions
+
+While Flux integrates with other systems it is built on Kubernetes core controller-runtime and properly adheres to Kubernetes security model including RBAC.
+
+<!-- See [Flux RBAC manifests](https://github.com/fluxcd/flux2/tree/main/manifests/rbac)
+- Scott: spell out here in high-level human-readable terms -->
+
+Flux installs a set of [RBAC manifests](https://github.com/fluxcd/flux2/tree/main/manifests/rbac).
+These include:
+
+1. A `crd-controller` `ClusterRole`, which:
+    - Has full access to all the Custom Resource Definitions defined by Flux controllers
+    - Can get, list, and watch namespaces and secrets
+    - Can get, list, watch, create, patch, and delete configmaps and their status
+    - Can get, list, watch, create, patch, and delete coordination.k8s.io leases
+2. A `crd-controller` `ClusterRoleBinding`:
+    - References `crd-controller` `ClusterRole` above
+    - Bound to a service accounts for every Flux controller
+3. A `cluster-reconciler` `ClusterRoleBinding`:
+    - References `cluster-admin` `ClusterRole`
+    - Bound to service accounts for only `kustomize-controller` and `helm-controller`
+
+Flux uses these two `ClusterRoleBinding` strategies in order to follow the principle of least privilege, and to allow for clear access separation using tools purpose built for policy enforcement (OPA, Kyverno, admission controllers).
+
+For example, the design allows all controllers to access Flux CRDs (binds to `crd-controller` `ClusterRole`), but only binds the Flux reconciler controllers for Kustomize and Helm to `cluster-admin` `ClusterRole`, as these are the only two controllers that manage resources in the cluster.
+
+However in a soft multi-tenancy setup, Flux does not reconcile a tenant's repo under the `cluster-admin` role.
+Instead you specify a different service account in your manifest, and the Flux controllers will use the Kubernetes Impersonation API under `cluster-admin` to impersonate that service account.
+In this way, policy restrictions for this service account are applied to the manifests being reconciled.
+If the binding is not defined for the correct service account and namespace, it will fail.
+The roles and permissions for this multi-tenancy approach are described in detail here: https://github.com/fluxcd/flux2-multi-tenancy
+
+## External audit
+
+- In 2021 there was a security assessment and fuzzer development of the FluxCD repositories made by Ada Logics and funded by the OSTIF.
+- For a summary, please see [this blog post](https://fluxcd.io/blog/2021-11-10-flux-security-audit/)
+- For all details, please read the [audit full technical report](https://fluxcd.io/FluxFinalReport-v1.1.pdf)
```

Comment:
I wonder if this section shouldn't be under `/security` as well?

Reply:
This is a good idea too 👍

Reply:
I agree with the rest but I think this is important for end users who are assessing how secure Flux is. I think the other page is for security researchers to know how the actual process works for reporting, handling, and disclosures etc

Reply:
My thinking would be to have a very visible `note` at the top which recommends having at look at `/security` for  up-to-date info on CVEs/audits and other security project initiatives.

And then have `/docs/security` be a page which is  just documentation about Flux security.

That way we avoid having to update information in multiple places.

Reply:
WDYT?

Reply:
Sure that works